components/dashboard/src/workspaces/Workspaces.tsx

```diff
@@ -98,75 +106,156 @@ const WorkspacesPage: FunctionComponent = () => {
+                                                            className="bg-red-50 dark:bg-red-300 hover:bg-red-100 dark:hover:bg-red-200 text-red-600 hover:text-red-700 hover:opacity-100"
+                                                            onClick={(evt) => {
+                                                                setDeleteModalVisible(true);
+                                                                evt.stopPropagation();
+                                                            }}
+                                                        >
+                                                            Delete Inactive Workspaces
+                                                        </Button>
+                                                    ) : null}
+                                                </div>
                                             </div>
-                                        </div>
-                                        <div className="self-center">
                                             {showInactive ? (
-                                                <Button
-                                                    variant="ghost"
-                                                    // TODO: Remove these classes once we decide on the new button style
-                                                    // Leaving these to emulate the old button's danger.secondary style until we decide if we want that style or not
-                                                    className="bg-red-50 dark:bg-red-300 hover:bg-red-100 dark:hover:bg-red-200 text-red-600 hover:text-red-700 hover:opacity-100"
-                                                    onClick={(evt) => {
-                                                        setDeleteModalVisible(true);
-                                                        evt.stopPropagation();
-                                                    }}
-                                                >
-                                                    Delete Inactive Workspaces
-                                                </Button>
+                                                <>
+                                                    {filteredInactiveWorkspaces.map((info) => {
+                                                        return <WorkspaceEntry key={info.id} info={info} />;
+                                                    })}
+                                                </>
                                             ) : null}
                                         </div>
+                                    )}
+                                </ItemsList>
+                            </div>
+                            {/* Show Educational if user is in gitpodIo */}
+                            {isGitpodIo() && (
```

Comment:
Is there no way to dismiss this side banner? It feels really intrusive considering there's bound to be many users who either already are knowledgeable about these topics or at least know our docs are also accessible via the account dropdown.

Reply:
Check this [slack message](https://gitpod.slack.com/archives/C074K0JASMQ/p1720545416829909) for context 🙏🏻